openerp/addons/test_new_api/models.py

```diff
@@ -143,12 +143,16 @@ class Discussion(models.Model):
     participants = fields.Many2many('res.users')
     messages = fields.One2many('test_new_api.message', 'discussion')
     message_changes = fields.Integer(string='Message changes')
-    important_messages = fields.One2many('test_new_api.message', 'discussion',
-                                         domain=[('important', '=', True)])
+    important_messages = fields.One2many(
+        'test_new_api.message', 'discussion',
+        domain=lambda self: self._domain_important())
     emails = fields.One2many('test_new_api.emailmessage', 'discussion')
     important_emails = fields.One2many('test_new_api.emailmessage', 'discussion',
                                        domain=[('important', '=', True)])
 
+    def _domain_important(self):
```

Comment:
Maybe you should indicate that the domain is put this way for trying the error or develop another separate test for this.